root/documentation/vep.conf

```
@@ -194,6 +194,11 @@
         description=LOFTEE identifies LoF (loss-of-function) variation. See <a target="_blank" href="https://github.com/konradjk/loftee/blob/master/README.md">README</a> for more details.
         default=0
       </LoF>
+      <UTRannotator>
+        type=Boolean
+        description=Annotates high-impact five prime UTR variants either creating new upstream ORFs or disrupting existing upstream ORFs. See <a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/main/UTRAnnotator.pm">README</a> for more details.
```

To keep consistent with other plugins. 

```suggestion
        description=Annotates high-impact five prime UTR variants either creating new upstream ORFs or disrupting existing upstream ORFs (<a target="_blank" href="https://raw.githubusercontent.com/ensembl-variation/VEP_plugins/main/UTRAnnotator.pm">plugin details</a>)
```